### ISSUE TITLE
"This extension may have been corrupted" - happening multiple times/week on two separate machines and two separate browsers

### ISSUE THREAD

**Issue by TriggerDingus** [brave/brave-browser#19573]
I posted this issue in Reddit but never received a response, though I saw at least one older post describing the same issue and nobody had responded to that one either. I'm convinced this issue isn't just me since I experience the issue:

On two separate Linux machines (Pop OS)
On two separate browsers on each of the machines (Chromium and Firefox)

At least once a week, I find my Bitwarden extension disappears from my browser extensions bar in each of the browsers on each of the machines. Upon investigation in the Manage Extensions page, I find the entry for Bitwarden says: "This extension may have been corrupted". In addition to the usual links for Details and Remove, there's a Repair link.

This issue began occurring a few months back and hasn't gotten any better or any worse. I don't know that I'm doing anything to trigger the behavior - I'm just using the browsers as usual.

**Comment by rebron** [comment]
@TriggerDingus Is this specific to Bitwarden? You brought up other browsers. We have an issue template and would prefer to have that filled out with more information, e.g. browser version, os, platform and a bit more detail.

**Comment by TriggerDingus** [comment]
Yes, this is only my Bitwarden extension(s) that get corrupted. I've not had an issue with any of my other extensions.

Here are the versions, though keep in-mind I update frequently and this issue has been occurring a few months.

Bitwarden extension 1.54.0
Pop OS: 21.04
Brave: Version 1.15.76 Chromium: 86.0.4240.111 (Official Build) (64-bit)
Firefox: 94.0.1

**Comment by TriggerDingus** [comment]
![BW Corrupt in Brave](https://user-images.githubusercontent.com/75340371/144352898-f9b0e50f-a161-449d-969b-8a10644aea35.jpg)

**Comment by curtvict** [comment]
This also happens to me often. Ubuntu 20.04, most recent versions of Chrome & the extension.

**Comment by TriggerDingus** [comment]
Still happening for me. Doesn't look like there's any movement on it?

**Comment by luv2dnce33** [comment]
Any updates here? Same problem on Brave for me. damn annoying...

**Comment by TriggerDingus** [comment]
> As an update, it seems to be happening much less often on Firefox - so much so that I would consider it no longer an issue on Firefox. I use Brave far less often, but I still see the issue there on a semi-regular basis.

I retract this. It's back just as often on Firefox.
This is a couple years on and appears to be common with others and not just me. Strange it hasn't been resolved yet.
Just as I responded back in 12/2021 this issue ONLY occurs with my Bitwarden extensions in both Firefox and Chromium. None of my other extensions have corrupted even once - it's only Bitwarden.

**Comment by luv2dnce33** [comment]
@bsclifton @atuchin-m anyone?

**Comment by luv2dnce33** [comment]
sad

**Comment by luv2dnce33** [comment]
god damn, this project is such a shitshow

**Comment by EmirLogas** [comment]
I also encountered this error today. I think doing [this](https://community.brave.com/t/bitwarden-extension/502298/6) solved it.
